### PR TITLE
GLT-3086: added run data approval fields

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: miso-web/src/main/webapp/scripts/form_run.js [hunk 1]
```diff
@@ -205,6 +207,47 @@ FormTarget.run = (function($) {
...
+          getSource: function() {
+            return [{
+              label: 'True',
+              value: true
+            }, {
+              label: 'False',
+              value: false
+            }];
+          },
+          convertToBoolean: true,
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/form_run.js]: What do you think about `Approved`/`Not Approved`?

Comment (djcooke) [reply miso-web/src/main/webapp/scripts/form_run.js]: For the dropdown labels you mean? "Data Approved: Not Approved" seems kind-of weird

Maybe "Yes"/"No" would be better than "True"/"False"?

Comment (callunity) [reply miso-web/src/main/webapp/scripts/form_run.js]: I like "Yes"/"No"!

**Comment by djcooke** [comment]
@callunity: updated

**Pull request merged**